cmd/gen-stdlib-allowlists/main.go

```
@@ -0,0 +1,106 @@
+//  2. `go env GOROOT` — the upstream-recommended replacement for
+//     `runtime.GOROOT()`; works on every machine where `go` is on PATH.
+//
+// Returns:
+//   - string: absolute path to GOROOT, or "" when both lookups fail.
+func detectGOROOT() string {
+	//: $GOROOT wins when set — bazel wrappers and `make generate` provide it.
+	if env := strings.TrimSpace(os.Getenv("GOROOT")); env != "" {
+		return env
+	}
+	//: Fall back to the upstream-recommended `go env GOROOT` invocation.
+	if out, err := exec.Command("go", "env", "GOROOT").Output(); err == nil {
```

golangci-lint (2.12.2)

[error] os/exec.Command must not be called. use os/exec.CommandContext

(noctx)